Game/MyProject/Game/Helper/HelperManager.cpp

```diff
@@ -0,0 +1,548 @@
+//-------------------------------------------------------------------------------
+//-- HelperManager.cpp ----------------------------------------------------------
+//-------------------------------------------------------------------------------
+#include"HelperManager.h"
+//-------------------------------------------------------------------------------
+//  PUBLIC METHODS
+//-------------------------------------------------------------------------------
+
+//-------------------------------------------------------------------------------
+// @ HelperManager::CreateBoxMesh()
+//-------------------------------------------------------------------------------
+std::shared_ptr<Mesh>
+HelperManager::CreateBoxMesh()
```

Comment:
this is not how you used indexed drawing. i'm counting 24 vertices and redundant indexing that i don't even think works. doing indexed drawing like this is actually slower than non indexed. a cube should have 8 vertices and a proper index buffer to generate the triangles from those 8 vertices.